download.jl

```diff
@@ -0,0 +1,82 @@
+#Extracting wind data from MET file 
+download("https://earthsciml.s3.amazonaws.com/compressed_Input/met/mcipv5/METDOT3D_160701.bson")
+
+#Change the name after downloading the file from AWS. 
+#The directories will vary by users 
+
+file = "/var/folders/s3/d0ks9vxn08dclbqqy0q3tgzw0000gn/T/jl_MRWMZx"
+mv(file, "METDOT3D_160701.bson")
+
+#Use decompress_netcdf and the change .bson file to .nc
+#The function is defined below
+#The directories will vary by users 
+comp_file = "/Users/Minwoo/Desktop/METDOT3D_160701.bson"
+output = "/Users/Minwoo/Desktop/METDOT3D_160701.nc"
+
+#Running a function
+decompress_netcdf(comp_file,output)
+
+
+
+#Extracting emission data from emis file 
+download("https://earthsciml.s3.amazonaws.com/compressed_Input/emis/gridded_area/gridded/emis_mole_all_20160701_cb6_bench.bson")
+
+#Change the name after downloading the file from AWS. 
+#The directories will vary by users
+file = "/var/folders/s3/d0ks9vxn08dclbqqy0q3tgzw0000gn/T/jl_MRWMZx"
+mv(file, "emis_mole_all_20160701_cb6_bench.bson")
+
+#Use decompress_netcdf and the change .bson file to .nc
+#The function is defined below
+#The directories will vary by users 
+comp_file = "/Users/Minwoo/Desktop/emis_mole_all_20160701_cb6_bench.bson"
+output = "/Users/Minwoo/Desktop/emis_mole_all_20160701_cb6_bench.nc"
+
+#Running a function
+decompress_netcdf(comp_file,output)
+
+
+
+using NetCDF,ZfpCompression,BSON
+
+"""
+    decompress_netcdf(oldfile,output)
+
+Decompress the bson file and create a new netcdf file withthe given values and variables.
+
+#Arguments
+- `comp_file::string`: the name of the compressed bson file.
+- `output::string`: the name of the new NetCDF file.
+"""
+
+#Decompression function
+function decompress_netcdf(comp_file,output)
```

Comment:
You shouldn't need to redefine this function, you can just do `include("decompress.jl")` to load the function from the other file.